chapter_preface/index_vn.md

```diff
@@ -667,30 +700,39 @@ Raju Gulabani, Charlie Bell, and Andrew Jassy for their generous support in writ
 * If you plan to run this code on GPUs, do not forget to install the necessary drivers and update your configuration.
 -->
 
-*dịch đoạn phía trên*
-
+* Học sâu đã cách mạng hóa nhận dạng khuôn mẫu, đưa ra công nghệ cốt lõi hiện được sử dụng trong nhiều ứng dụng công nghệ, bao gồm thị giác máy, xử lý ngôn ngữ tự nhiên, nhận dạng giọng nói tự động.
+* Để áp dụng thành công kĩ thuật học sâu, bạn phải hiểu được cách đặt vấn đề, mô hình hóa dạng toán, các thuật toán để khớp mô hình theo dữ liệu của bạn, và các kỹ thuật để thực hiện tất cả những điều này.
```

Comment:
Từ "cast" mang nghĩa là "chuyển đổi" hoặc "biến đổi". Ý của câu này là phải "chuyển đổi bài toán"
```suggestion
* Để áp dụng thành công kĩ thuật học sâu, bạn phải hiểu được cách biến đổi bài toán, toán học của việc mô hình hóa, các thuật toán để khớp mô hình theo dữ liệu của bạn, và các kỹ thuật để thực hiện tất cả những điều này.
```